### PR TITLE
don't include central ligand self edge in radial networks

### PR DESCRIPTION
**Pull request by richardjgowers**
fixes issue #544

## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

### CODE REVIEW

File: openfe/setup/ligand_network_planning.py [hunk 1]
```diff
@@ -84,6 +85,8 @@ def generate_radial_network(
     edges = []
 
     for ligand in ligands:
+        if ligand == central_ligand:
```
Comment (mikemhenry) [inline openfe/setup/ligand_network_planning.py:88]: No warning? Or is this something that a user would expect we handle in this way? I think it makes sense to skip it this way the more I think about it

**Comment by richardjgowers** [comment]
I documented the behaviour, so arguably its not a warning any more. It does
feel easier to not have to manually separate out the central ligand and
just chuck them in.

File: openfe/setup/ligand_network_planning.py (hunk 1 above)
Comment (IAlibay) [reply openfe/setup/ligand_network_planning.py:88]: Probably would second that.

**Review by IAlibay — APPROVED** [review]
I'll approve but a warning message would be good.

**Review by hannahbaumann — APPROVED** [review]

**Pull request merged**